### ISSUE TITLE
Maybe an iconic font instead of images?

### ISSUE THREAD

**Issue by stevdompl** [drgullin/icheck#332]
What I would like to see in iCheck is the use of iconic fonts (woff, ttf/otf, svg, eot) rather than images. 

This would make the theming much easier:
- as the style would be achieved by using different characters sets in the font,
- as the color would inherit from text color,
- and as fonts render as vector there (I think) there would be no reason for Retina variants.

**Comment by custa1200** [comment]
Seems like a lot of overkill for some checkboxes and radio buttons. I thought people had also moved on from using fonts due to various issues and pushing SVG images.

**Comment by stevdompl** [comment]
I don't think so. For the use of CDN, we could even split the styles in separate fonts, so the browser would download only required files. You should also keep in mind that the browser can cache the fonts (which IMO will need less disk space than storing X images - considering the use of the combined stylesheet).

The reason behind this issue is that I would like to use iCheck in one of my projects, but because I allow the user to change Bootstrap theme by choosing one of [Bootswatch variants](http://bootswatch.com/) I'm forced to prepare additional color variants to suit the used color scheme.

Of course, I don't insist on font usage. We could also do it with some HTML tags, pseudo-elements, and CSS3 transforms, and compile along with the Bootstrap theme or use of `currentColor` to define background color - but this would mean a lower browser compatibility.

**Comment by vernondegoede** [comment]
I agree with @stevdom, using images instead of iconic fonts makes theming of this plugin harder. 

As an example, I'm trying to let my users style a certain page using a colorpicker. Everything is styleable using CSS except for the iCheck radio buttons...

**Comment by stevdom** [comment]
You're right. That's why I'm using now (only checkbox & radio) modified styles from http://fezvrasta.github.io/bootstrap-material-design/#checkbox which is based on box-shadow and with the use of _currentColor_ and _text-(success|info|warning|danger) classes_ it behaves as expected.

Thanks for the replies.

**Comment by stevdom** [comment]
Well actually I'm using right now modified (only checkbox & radio) styles from [here](http://fezvrasta.github.io/bootstrap-material-design/#checkbox) which are based on box-shadow.

**Comment by stevdompl** [comment]
Well actually I'm using right now modified (only checkbox & radio) styles from [here](http://fezvrasta.github.io/bootstrap-material-design/#checkbox) which are based on _box-shadow_.
